examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
```

Comment:
if we want to have one image supporting multiple scans, we need to think about adding more orchestration in the entrypoint (like have supervising process that run clam/etc)... agree, that is bigger discussion.